js/renovation/ui/scroll_view/scrollable_native.tsx

```diff
@@ -171,11 +176,12 @@ export class ScrollableNative extends JSXComponent<ScrollableInternalPropsType>(
   }
 
   get cssClasses(): string {
-    const { direction, classes } = this.props;
+    const { direction, classes, disabled } = this.props;
 
     const classesMap = {
-      'dx-scrollable dx-scrollable-native dx-scrollable-renovated dx-scrollable-native-generic': true,
+      [`dx-scrollable dx-scrollable-native dx-scrollable-native-${devices.current().platform} dx-scrollable-renovated`]: true,
```

Comment:
Why 'devices.current().platform' while it was 'devices.real().platform'?

```
        const device = **devices.real();**
        const deviceType = device.platform;

        this._$element
            .addClass(SCROLLABLE_NATIVE_CLASS)
            .addClass(SCROLLABLE_NATIVE_CLASS + '-' + deviceType)
```

Reply:
Yep, my mistake. Fixed https://github.com/DevExpress/DevExtreme/pull/15636/commits/9d8f0a12a9faf520c78e6ccdffcab5523984d43c